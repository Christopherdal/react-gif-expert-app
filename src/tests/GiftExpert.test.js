import React from 'react';
// import { render } from '@testing-library/react';
import { shallow } from 'enzyme';
import { ViewGiftExpert } from './../views/GiftExpert.view';

describe('Test on GiftExpert Component', () => {
	let wrapper = shallow(<ViewGiftExpert />);

	test('should Match With Snapshot GiftExpert Correctly', () => {
		expect(wrapper).toMatchSnapshot(); // To Save the literal rendered component and compare with wrapper if you press 'u' update the snapshot
	});
});
