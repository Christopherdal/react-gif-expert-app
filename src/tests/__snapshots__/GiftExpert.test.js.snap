// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on GiftExpert Component should Match With Snapshot GiftExpert Correctly 1`] = `
<Fragment>
  <h1>
    GiftExpertApp
  </h1>
  <UiAddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="One punch"
      key="One punch0"
    />
  </ol>
</Fragment>
`;
